packages/components/ember-cli-build.js

```diff
@@ -9,8 +9,12 @@ module.exports = function (defaults) {
       precision: 4,
       includePaths: [
         '../../node_modules/@hashicorp/design-system-tokens/dist/products/css',
+        '../../node_modules/ember-power-select/vendor',
       ],
     },
+    'ember-power-select': {
+      theme: false,
```

Comment:
Do apps need to do this in their `ember-cli-build.js` too? If so, could this be added to the docs?

Reply:
As far as I can tell, this is [the default configuration](https://github.com/hashicorp/cloud-ui/blob/master/hcp/ember-cli-build.js#L147-L149), but yes it's worth mentioning that these styles assume you use `ember-power-select` with the default theme.